README.md

```diff
@@ -2,13 +2,14 @@
 
 These images provide all of the required dependencies for running Cypress in Docker.
 
-We build three main images, click on the image name to see the available tags and versions.
+We build four images, click on the image name to see the available tags and versions.
 
-| Image                        | Default                     | Description                                                                       | Monthly pulls                                                                                                                         |
-| ---------------------------- | --------------------------- | --------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
-| [cypress/base](base)         | `cypress/base:16.13.0`      | All operating system dependencies, no Cypress, and no browsers.                   | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/base.svg?maxAge=604800)](https://hub.docker.com/r/cypress/base/)         |
-| [cypress/browsers](browsers) | `cypress/browsers:chrome69` | All operating system dependencies and some browsers.                              | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/browsers.svg?maxAge=604800)](https://hub.docker.com/r/cypress/browsers/) |
-| [cypress/included](included) | `cypress/included:9.4.1`    | All operating system dependencies, Cypress, and some browsers installed globally. | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/included.svg?maxAge=604800)](https://hub.docker.com/r/cypress/included/) |
+| Image                                                          | Default                     | Description                                                                        | Monthly pulls                                                                                                                         |
+| -------------------------------------------------------------- | --------------------------- | ---------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [cypress/factory](https://hub.docker.com/r/cypress/factory/)   | `cypress/factory:1.0.0`    | A template base image which can be used with ARGS to create a docker image with custom versions. | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/factory.svg?maxAge=604800)](https://hub.docker.com/r/cypress/factory/)   |
```

Comment:
I felt like this description could be a little streamlined for clarity. Perhaps something like...

```suggestion
| [cypress/factory](https://hub.docker.com/r/cypress/factory/)   | `cypress/factory:1.0.0`    | A base image template which can be used with ARGs to create a custom docker image. | [![Docker Pulls](https://img.shields.io/docker/pulls/cypress/factory.svg?maxAge=604800)](https://hub.docker.com/r/cypress/factory/)   |
```